lms/envs/aws.py

```diff
@@ -119,8 +121,8 @@
     """
 
     'collect_stats': {
-    'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
-    'schedule': 30, # number in seconds.
+        'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
+        'schedule': crontab(hour=0, minute=random.randint(1, 59))
```

Comment:
@z4y4ts, how I know, if admin do not want to share instance statistics (Paranoid+ level) he needs to comment or delete `CELERYBEAT_SCHEDULE` settings above. If I move schedule declaration to code directly (`tasks.py`), it will make admin way less attractive. What do you think?